eng/pipelines/maintenance-packages.yml

```diff
@@ -96,7 +96,46 @@ stages:
               $(_InternalBuildArgs)
             displayName: $(_BuildJobDisplayName)
 
+- stage: pack
+  displayName: Pack
+  jobs:
+  - template: /eng/common/templates/jobs/jobs.yml
+    parameters:
+      artifacts:
+        publish:
+          artifacts: false
+          logs: true
+          manifests: true
+      enableMicrobuild: true
+      enablePublishUsingPipelines: false
+      publishAssetsImmediately: false
+      enablePublishTestResults: false
+      enableSourceIndex: ${{ and(eq(variables._RunAsInternal, True), eq(variables['Build.SourceBranch'], 'refs/heads/main')) }}
+      workspace:
+        clean: all
+      jobs:
+      - ${{ if eq(variables._RunAsPublic, True) }}:
+        - job: linux
```

Comment:
I suspect we only need one OS job to verify this. Do we need to also test it in Windows? Would we get different ApiCompat errors?

Reply:
Linux should be sufficient and the errors should be the same.